quail/processes/wps_climdex_sdii.py

```diff
@@ -0,0 +1,124 @@
+import os
+from rpy2 import robjects
+from pywps import Process, LiteralInput
+from pywps.app.Common import Metadata
+
+from wps_tools.utils import log_handler, collect_args, common_status_percentages
+from wps_tools.io import log_level
+from quail.utils import get_package, logger, load_rdata_to_python, save_python_to_rdata
+from quail.io import climdex_input, ci_name, output_file, rda_output, vector_name
+
+
+class ClimdexSDII(Process):
+    """
+    computes the climdex index SDII, or Simple Precipitation Intensity Index.
+    This isdefined as the sum of precipitation in wet days (days with
+    preciptitation over 1mm) during the year divided by the number of wet days
+    in the year.
+    """
+
+    def __init__(self):
+        self.status_percentage_steps = dict(
+            common_status_percentages,
+            **{
+                "load_rdata": 10,
+                "save_rdata": 90,
+            },
+        )
+        inputs = [
+            climdex_input,
+            ci_name,
+            output_file,
+            vector_name,
+            log_level,
+        ]
```

Comment:
I have never seen a list of PyWPS inputs this clean. :+1:

Reply:
I second that!